src/interfaces/WF/WFIPAddressType.ts

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
Could this file be renamed to `WFIPAddressTypeOption` please? Just to keep it consistent with the name of the type it defines.